_posts/2020-02-03-release-2.0.0.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Release v2.0.0
+tags: ["v2.0.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 03/02 2020 the new version 2.0.0 is out
+<!--more-->
+
+This major release was focused on restructuring the project, its artifacts, and configurations. 
+These are not changes that should affect users but are breaking for the people who deploy the project.
+
+## Breaking Changes
+- [#64]({{ site.github.issues_url }}/64) Standardization and Conformance are now combined in one artifact - `spark-jobs.jar`. 
+This allows using a single JAR file for both jobs. 
+The class names for the jobs remain the same. 
+*Note:* This change requires updating helper scripts and setting the `SPARK-JOBS-JAR` environment variable in `enceladus_env.sh`.
+- [#1090]({{ site.github.issues_url }}/1090) All `Enceladus` modules are now under the same `enceladus` group ID in the `absa.co.za` organization. 
+They will no longer have the `enceldus-` prefix and will not appear as top-level artifacts in maven.
+- [#1099]({{ site.github.issues_url }}/1099) `Menas` properties no longer have the `za.co.absa.encealdus.` prefix. 
+See the `application.properties.template` fo details in deploying Menas.
+
+## Conformance Improvements
+- [#1078]({{ site.github.issues_url }}/1078) Preliminary work on an improved broadcast strategy for the mapping conformance rule. 
+It can now join fields located at different array levels. 
+This mapping rule strategy is planned to be turned on in experimental mode in Enceladus v.3.0.0.
+- [#1129]({{ site.github.issues_url }}/1129) The broadcasting mapping rule strategy can now be used in experimental mode. 
+To do this set `conformance.mapping.rule.broadcast=auto` as the option to run dynamic conformance. 
+The `conformance.mapping.rule.max.broadcast.size.mb` option specifies the maximum mapping table size that is allowed to be broadcast.
+- [#1136]({{ site.github.issues_url }}/1136) Nested array transformations API were extracted into a separate project: [spark-hats](https://github.com/AbsaOSS/spark-hats).
+- [#1159]({{ site.github.issues_url }}/1159) Fixed the broadcasting mapping rule dropping of standardization errors when trying to conform a null array.
+
+## Standardization Improvements
+- [#1101]({{ site.github.issues_url }}/1101) Added support for metadata in arrays of simple types (e.g. `pattern` can be specified for array of timestamps).
+
+## Menas Improvements
+- [#1118]({{ site.github.issues_url }}/1118) Update `Spring Boot` to version `2.2.2`.
+- [#1151]({{ site.github.issues_url }}/1151) Fixed issue with selecting the `Runs` tile on the landing page not loading the `Runs` in the navigation.
+- [#1154]({{ site.github.issues_url }}/1154) Merge of Standardization and Conformance is not reflected in Oozie.
```

Comment:
I think remove this line. It's a bugfix of one of the main features of this release, fixed before made the release went out. No reason to mention it, IMO.